wagtail/wagtailadmin/utils.py

```diff
@@ -14,6 +15,9 @@
 from wagtail.wagtailusers.models import UserProfile
 
 
+logger = logging.getLogger('wagtail.wagtailadmin')
```

Comment:
Can the logger be called ``wagtail.admin`` (to make it consistent with [wagtail.core](https://github.com/torchbox/wagtail/blob/master/wagtail/wagtailcore/models.py#L47) and [wagtail.frontendcache](https://github.com/torchbox/wagtail/blob/master/wagtail/contrib/wagtailfrontendcache/backends.py#L11))